ai-machine-learning/thoth-thamos/advise.md

```diff
@@ -0,0 +1,49 @@
+Thamos CLI runs based on the `.thoth.yaml` config, which we have setup already in the example repository here. 
+
+You can browse the config file in the editor or via terminal using -
+
+``cat .thoth.yaml``{{execute}} 
```

Comment:
I think you can do something like `filename.yaml`{{open}}, and it should create a hyperlink that directly opens the file in the editor. Just a suggestion might make it look cleaner

Reply:
Somehow it seems to only work on root directory. Like doesn't look for the file in subfolders. Do you have to put a full path link?

Reply:
I don't know, could you test it and lmk?

Reply:
@4n4nd I have updated the file links :link: